src/basic/ordered-set.h

```diff
@@ -62,6 +62,16 @@ static inline char **ordered_set_get_strv(OrderedSet *s) {
         return _hashmap_get_strv(HASHMAP_BASE((OrderedHashmap*) s));
 }
 
+static inline int ordered_set_ensure_put(OrderedSet **s, const struct hash_ops *ops, void *p) {
+        int r;
+
+        r = ordered_set_ensure_allocated(s, ops);
```

Comment:
Please pass allocation info here similarly to `_ordered_hashmap_ensure_put`. In general such additions should be a separate commit.